test/util/testnode/full_node.go

```diff
@@ -168,7 +168,10 @@ func NewNetwork(t testing.TB, cfg *Config) (cctx Context, rpcAddr, grpcAddr stri
 		genState = opt(genState)
 	}
 
-	chainID := tmrand.Str(6)
+	chainID := cfg.ChainID
+	if chainID == "" {
+		chainID = tmrand.Str(6)
+	}
```

Comment:
this was a good idea @Frenzoid, but using a string of "" is valid, so we can just leave it up to the users on this one

all we have to do is blindly use the chain-id in the config